CommunicationLayer/CommDeviceControl/UdpConnectionService.cpp

```diff
@@ -1,49 +0,0 @@
-#include <QUdpSocket>
```

Comment:
I think this file will still be necessary, we will be connecting to the server using Udp, but it is true the logic in here doesn't make sense. I think what we should do is leave the file but turn everything into stubs.